core/src/main/java/com/scalar/db/api/TableMetadata.java

```diff
@@ -162,7 +163,8 @@ public boolean equals(Object o) {
         && Objects.equals(partitionKeyNames, metadata.partitionKeyNames)
         && Objects.equals(clusteringKeyNames, metadata.clusteringKeyNames)
         && Objects.equals(clusteringOrders, metadata.clusteringOrders)
-        && Objects.equals(secondaryIndexNames, metadata.secondaryIndexNames);
+        && Objects.equals(secondaryIndexNames, metadata.secondaryIndexNames)
+        && Objects.equals(encryptedColumnNames, metadata.encryptedColumnNames);
```

Comment:
I forgot to add `encryptedColumnNames` to `equals()` and `hashCode()` in the previous PR...